app/javascript/controllers/toggle_controller.js

```diff
@@ -21,12 +21,20 @@ export default class extends Controller {
       this.hasCloseClass && element.classList.remove(this.closeClasses)
       this.hasOpenClass && element.classList.add(this.openClasses)
     })
+    this.chevronTargets.forEach(element =>{
+      element.classList.remove("rotate-0")
```

Comment:
If we remove `rotate-0` from the HTML then we can only worry about `rotate-180`.

Also, this class should be part of the `classes` static variable at the top of this file and set in the HTML.